dark/utils.py

```diff
@@ -95,3 +99,25 @@ def median(l):
         raise ValueError('arg is an empty sequence')
     else:
         return _median(l)
+
+
+@contextmanager
+def asHandle(handleish, mode='r'):
+    """
+    Decorator for file opening that makes it easy to open compressed files.
+    Based on L{Bio.File.as_handle}.
+
+    @param handleish: Either a C{str} or a file handle.
```

Comment:
That's taken from Bio.File.  In Python people often append `ish` to variable names to indicate that something may not be of that exact type, as long as it can be treated that way (e.g., `fileish` for something that behaves like a file, even though it might be a `StringIO` instance). Anyway, I changed the name to `fileNameOrHandle` which is a lot clearer.